index.php

```diff
@@ -1,4 +1,243 @@
+                <path fill-rule="evenodd" d="M2.25 4.5A.75.75 0 0 1 3 3.75h14.25a.75.75 0 0 1 0 1.5H3a.75.75 0 0 1-.75-.75Zm0 4.5A.75.75 0 0 1 3 8.25h9.75a.75.75 0 0 1 0 1.5H3A.75.75 0 0 1 2.25 9Zm15-.75A.75.75 0 0 1 18 9v10.19l2.47-2.47a.75.75 0 1 1 1.06 1.06l-3.75 3.75a.75.75 0 0 1-1.06 0l-3.75-3.75a.75.75 0 1 1 1.06-1.06l2.47 2.47V9a.75.75 0 0 1 .75-.75Zm-15 5.25a.75.75 0 0 1 .75-.75h9.75a.75.75 0 0 1 0 1.5H3a.75.75 0 0 1-.75-.75Z" clip-rule="evenodd" />
+            </svg>
+        </button>
+    </nav>
+    <main class="group grow h-screen">
+        <section class="group-[.minimised]:h-[calc(100%-6rem)] h-3/4 p-12 overflow-auto">
+            <h2 class="text-4xl font-bold">Home</h2>
+            <div class="grid md:grid-cols-2 lg:grid-cols-3 mt-12 gap-5">
+                <div class="">
+                    <h3 class="text-xl font-bold mb-3">Artists</h3>
+                    <div class="grid grid-cols-2 gap-3 mb-3">
+                       <?php
+                        foreach ($artists as $artist) {
+                            $artistId = $artist->getId();
+                            $artistAlbumCount = $artistsModel->getArtistsAlbumCount($artistId);
+                            $aristAlbumArtworks = $artistsModel->getArtistAlbumArtworks($artistId);
+                            $artworkDisplay = '';
+                            foreach ($aristAlbumArtworks as $aristAlbumArtwork){
+                                $artworkDisplay .= "<img src={$aristAlbumArtwork->getArtworkUrl()} />";
+                            }
+                            echo "
+                                 <a href='artist.php?{$artistId}' class='rounded bg-cyan-950 p-3 hover:bg-cyan-800 hover:cursor-pointer'>
+                                    <div class='flex gap-2 h-8'>
+                                        {$artworkDisplay}
+                                    </div>
+                                    <h4 class='text-xl font-bold'>{$artist->getArtistName()}</h4>
+                                    <p>{$artistAlbumCount->getAlbumCount()} Albums</p>
+                                </a>
+                            ";
+                        }
+                        ?>
+                        <div class="rounded bg-cyan-950 p-3 flex items-center">
+                            <h4 class="text-2xl text-slate-500">+ 16 more</h4>
+                        </div>
+                    </div>
+                    <a href="artists.php" class="float-right border rounded-md bg-cyan-950 px-2 py-1 hover:bg-cyan-800">See all
+                        <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke-width="1.5" stroke="currentColor" class="size-6 inline">
+                            <path stroke-linecap="round" stroke-linejoin="round" d="m12.75 15 3-3m0 0-3-3m3 3h-7.5M21 12a9 9 0 1 1-18 0 9 9 0 0 1 18 0Z" />
+                        </svg>
+                    </a>
+                </div>
+<!--                <div class="">-->
```

Comment:
Now you should remove all this comment out stuff. Commented out HTML still gets sent to the browser, so it'll be slowing down page load for no reason